src/main/java/org/prebid/server/bidder/grid/model/ExtImpGrid.java

```diff
@@ -1,11 +1,20 @@
 package org.prebid.server.bidder.grid.model;
 
-import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Value;
+import org.prebid.server.proto.openrtb.ext.request.ExtImpPrebid;
+import org.prebid.server.proto.openrtb.ext.request.grid.ExtImpGridBidder;
 
-@AllArgsConstructor(staticName = "of")
 @Value
+@Builder(toBuilder = true)
 public class ExtImpGrid {
```

Comment:
Why this class not in `org.prebid.server.proto.openrtb.ext.request.grid` package?
Same for all related classes.